src/__tests__/actions/update_comment.action.test.js

```diff
@@ -0,0 +1,51 @@
+import thunkMiddleware from 'redux-thunk';
+import axios from 'axios';
+import promiseMiddleware from 'redux-promise-middleware';
+import configureStore from 'redux-mock-store';
+import MockAdapter from 'axios-mock-adapter';
+import {
+  updating, successfulUpdate, failedUpdate, updateComment,
+} from '../../actions/update_comment.action';
+
+const middlewares = [
+  thunkMiddleware,
+  promiseMiddleware(),
+];
+
+const mockStore = configureStore(middlewares);
+const store = mockStore({});
+describe('test updating a comment action', () => {
+  afterEach(() => {
+    store.clearActions();
+  });
+
+  it('loads on dispatching updating action', () => {
+    store.dispatch(updating());
+    const expectedOutput = [{ type: 'UPDATE_COMMENT_LOADING' }];
+    expect(store.getActions()).toEqual(expectedOutput);
+  });
+
+  it('successfulUpdate gets dispatched appropriately', () => {
+    store.dispatch(successfulUpdate('data'));
+    const expectedOutput = [{ data: 'data', type: 'UPDATE_COMMENT_SUCCESS' }];
+    expect(store.getActions()).toEqual(expectedOutput);
+  });
+
+  it('dispatches an error on fail', () => {
+    store.dispatch(failedUpdate(['error']));
+    const expected = [{ errors: ['error'], type: 'UPDATE_COMMENT_FAILURE' }];
+    expect(store.getActions()).toEqual(expected);
+  });
+
+  it('calls update comment', async () => {
+    const mock = new MockAdapter(axios);
+    const data = {
+      comment: {
+        body: 'new comment',
+      },
+    };
+    mock.onPut('http://127.0.0.1:8000/api/articles/hi-there/comments/1/', data).reply(200, { data });
```

Comment:
Chane this to a relative url `http://127.0.0.1:8000/api/articles/hi-there/comments/1/`

Reply:
ok. fixing this